common/views/components/IIIFViewerPrototype/ViewerSidebarPrototype.tsx

```diff
@@ -252,9 +255,9 @@ const ViewerSidebarPrototype: FunctionComponent<Props> = ({
             </p>
             {credit && (
               <p>
-                <a href="https:wellcomecollection.org/works/{work.id}">
-                  {credit}
-                </a>
+                <NextLink href={`/works/${work.id}`}>
```

Comment:
Could we use the `WorkLink`? We should probably add a source then too.

Sorry to be such a massive pedant on this.

Reply:
Yeah, I didn't use it because I couldn't see a relevant `source`. What are the sources for, and what should this one be, then?

Reply:
It's used in tracking. Basically we would able to see how many pageviews to work pages were generated by this component.